<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccountWidget</class>
 <widget class="QDialog" name="CreateAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creer Un Compte</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>VEUILLEZ REMPLIR LES CHAMPS SUIVANTS :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nomLabel">
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nomLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="prenomLabel">
     <property name="text">
      <string>Prenom:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="prenomLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ageLabel">
     <property name="text">
      <string>Age: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ageLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="adresseLabel">
     <property name="text">
      <string>Adresse:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="adresseLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="telLabel">
     <property name="text">
      <string>Telephone:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="telLineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="soldeLabel">
     <property name="text">
      <string>Solde Initial</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="soldeLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type De Compte:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="typeComboBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>Etudiant (e)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fonctionnaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sans-Travail</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ValiderButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AnnulerButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
